api/goldens/aptos_api__tests__events_test__test_get_events.json

```diff
@@ -1,10 +1 @@
-[
-  {
-    "key": "0x0400000000000000000000000000000000000000000000000000000000000000000000000a550c18",
```

Comment:
https://github.com/aptos-labs/aptos-core/blob/33c2d745f1d398310d3ffbd3e9eb5954a534ad18/api/src/tests/events_test.rs#L8

need to change this to 0x05 too

Reply:
Nice catch thanks!

Reply:
wtf? haha